### PR TITLE
tagging SmokeTest

### PR DESCRIPTION
**Pull request by aweng98**
Google doc https://docs.google.com/document/d/1wc3YWjwap3moDMSgcZ2jn5pKMeo2gfRBr2Lj0HS4HlE/edit#heading=h.4eebtq1k72he

Not new tests. Just a few tests that should run to ensure the firecloud-ui still works. Jenkins `firecloud-ui-fiab-test-runner` should finish much quicker after merge this PR.

### CODE REVIEW

**Comment by aweng98** [comment]
@davidangb @rtitle Do you think tagged tests are right set of firecloud-ui tests to run? that provides basic firecloud-ui coverage.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala [hunk 1]
```diff
@@ -6,12 +6,13 @@ import org.broadinstitute.dsde.workbench.auth.AuthToken
...
 import org.scalatest.{FreeSpec, Matchers, ParallelTestExecution}
 
 class DataLibrarySpec extends FreeSpec with ParallelTestExecution with WebBrowserSpec with UserFixtures with WorkspaceFixtures
   with BillingFixtures with Matchers with TestReporterFixture {
 
-  "For a dataset with consent codes" in {
+  "For a dataset with consent codes" taggedAs Tags.SmokeTest in {
     val curatorUser = UserPool.chooseCurator
     implicit val authToken: AuthToken = curatorUser.makeAuthToken()
     withCleanBillingProject(curatorUser) { billingProject =>
```
Comment (davidangb) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala:18]: feels like this test and the pre-existing smoketest in `PublishSpec` could be combined.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala [hunk 2]
```diff
@@ -52,7 +53,7 @@ class BillingSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Cl
          *
          * E comes last because it makes the project unusable.
          */
-        "should be able add a new user, create a workspace, and run a method, change billing account" in {
+        "should be able add a new user, create a workspace, and run a method, change billing account" taggedAs Tags.SmokeTest in {
```
Comment (davidangb) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala:56]: this feels like a superset of the existing smoketest in `WorkspaceOwnerSpec`  ...  can that other one be turned off?

The other test validates workspace creation with a gpalloc project, while this one validates workspace creation with a new project. We *may* want to keep both to isolate issues, but that will increase duration for the smoke tests.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceWriterSpec.scala [hunk 3]
```diff
@@ -26,7 +27,7 @@ class WorkspaceWriterSpec extends FreeSpec with Matchers with WebBrowserSpec wit
 
   "A user" - {
     "who has writer access" - {
-      "should only see the estimated monthly storage fee in the Project Cost section of the summary page" in {
+      "should only see the estimated monthly storage fee in the Project Cost section of the summary page" taggedAs Tags.SmokeTest in {
```
Comment (davidangb) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceWriterSpec.scala:30]: is this really worthwhile as a UI smoketest?

At the API level, we can test whether or not the APIs return costs and billing links, etc. That just leaves this test asserting that the UI properly displays what is returned from the APIs. That's valid, but is it worth it?

**Review by davidangb — COMMENTED** [review]
some comments inline. additionally:
* is the `AuthDomainGroupSpec` smoke test really testing anything that isn't already tested at the api level? That one doesn't feel necessary as a UI smoke test.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceWriterSpec.scala (hunk 3 above)
Comment (aweng98) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceWriterSpec.scala:30]: you're right about the reason. I picked this one to assert UI can display cost information. This is a relatively new UI feature. I think user probably cares about the cost, so I think it was something that should continue to be tested. Tests are run in parallel, wouldn't affect playback time much at all.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala (hunk 2 above)
Comment (aweng98) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala:56]: 👍 removed `WorkspaceOwnerSpec` smoketest.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala (hunk 1 above)
Comment (aweng98) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala:18]: 👍 removed `PublishSpec` smoketest. added `page should not (be(None))` check in this test.

**Comment by aweng98** [comment]
`AuthDomainGroupSpec` test were chosen for: workspace clone modal, authdomain groups can display in UI. both are not testable by api tests.

**Comment by aweng98** [comment]
jenkins retest

**Comment by aweng98** [comment]
@davidangb Thank you for reviewing PR. I think I responded all feedback. Any other comments before give thumbs-up?

**Review by davidangb — APPROVED** [review]
:+1: wfm!

**Pull request merged**